models/core/txs.yml

```diff
@@ -0,0 +1,14 @@
+
+version: 2
+
+models:
+  - name: txs
```

Comment:
probably can leave out the tx one given @mattromano is working on it in a separate PR

Reply:
Agree, I'll delete that